stores/moderationScreenStore.ts

```diff
@@ -10,11 +10,15 @@ export enum ModerationScreen {
 export const useModerationScreenStore = defineStore('moderationScreenStore', () => {
     const runtimeConfig = useRuntimeConfig();
     const activeScreen: Ref<ModerationScreen> = ref(ModerationScreen.dashboard)
-    const enableModerationPanel: Ref<boolean> = ref(runtimeConfig.public.ENABLE_MODERATION_PANEL || false)
+    const enableModerationPanel: Ref<boolean> = ref(false)
 
     function setActiveScreen(newValue: ModerationScreen) {
         activeScreen.value = newValue
     }
 
-    return { activeScreen, enableModerationPanel, setActiveScreen }
+    function setEnableModerationPanelToTrue (currentPath:string){  
+        runtimeConfig.public.ENABLE_MODERATION_PANEL && currentPath.includes("moderation") ? enableModerationPanel.value = true : null
```

Comment:
Could we avoid using `runtimeConfig` and just use `process.env.ENABLE_MODERATION_PANEL` directly? That would make testing way easier.